<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Modify</class>
 <widget class="QDialog" name="Modify">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QFrame" name="frame">
     <property name="styleSheet">
      <string notr="true">border-image: url();
border-image: url(:/new/pictures/pictures/10.png);</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>32</x>
        <y>28</y>
        <width>42</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url();
font: 10pt &quot;黑体&quot;;</string>
      </property>
      <property name="text">
       <string>单词：</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_3">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>28</y>
        <width>100</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url();
background-color: qlineargradient(spread:pad, x1:1, y1:0.5, x2:0, y2:0.517, stop:0 rgba(171, 128, 83, 0), stop:1 rgba(255, 255, 255, 255));</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>32</x>
        <y>54</y>
        <width>42</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url();
font: 10pt &quot;黑体&quot;;</string>
      </property>
      <property name="text">
       <string>词性：</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>32</x>
        <y>80</y>
        <width>42</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url();
font: 10pt &quot;黑体&quot;;</string>
      </property>
      <property name="text">
       <string>释义：</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="textEdit_3">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>80</y>
        <width>371</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url();
background-color: qlineargradient(spread:pad, x1:1, y1:0.5, x2:0, y2:0.517, stop:0 rgba(171, 128, 83, 0), stop:1 rgba(255, 255, 255, 255));</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>32</x>
        <y>150</y>
        <width>42</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url();
font: 10pt &quot;黑体&quot;;</string>
      </property>
      <property name="text">
       <string>例句：</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="textEdit_4">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>150</y>
        <width>371</width>
        <height>81</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url();
background-color: qlineargradient(spread:pad, x1:1, y1:0.5, x2:0, y2:0.517, stop:0 rgba(171, 128, 83, 0), stop:1 rgba(255, 255, 255, 255));</string>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>240</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url();
font: 11pt &quot;方正姚体&quot;;
background-color: qlineargradient(spread:pad, x1:1, y1:0.5, x2:0, y2:0.517, stop:0 rgba(171, 128, 83, 0), stop:1 rgba(255, 255, 255, 255));</string>
      </property>
      <property name="text">
       <string>修改</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_4">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>54</y>
        <width>280</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url();
background-color: qlineargradient(spread:pad, x1:1, y1:0.5, x2:0, y2:0.517, stop:0 rgba(171, 128, 83, 0), stop:1 rgba(255, 255, 255, 255));</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_3">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>0</y>
        <width>40</width>
        <height>40</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url();
border-image: url(:/new/pictures/pictures/14.png);
</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>lineEdit_3</tabstop>
  <tabstop>lineEdit_4</tabstop>
  <tabstop>textEdit_3</tabstop>
  <tabstop>textEdit_4</tabstop>
  <tabstop>pushButton_2</tabstop>
  <tabstop>pushButton_3</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_3</sender>
   <signal>clicked()</signal>
   <receiver>Modify</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>465</x>
     <y>19</y>
    </hint>
    <hint type="destinationlabel">
     <x>500</x>
     <y>23</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
